### ISSUE TITLE
Officially recommend an icon set

### ISSUE THREAD

**Issue by geoffrey-eisenbarth** [bigskysoftware/missing#92]
> Missing.css has classes like` .iconbutton`, but no icons. The example in the docs just uses aria-hidden Unicode characters. Find a good icon set that works well with Missing, and mention it in the docs. - [Missing Roadmap](https://cloud.dz4k.dev/s/mznkdLLiLnf27HE)

> https://lucide.dev/ this is the best (most comprehensive, easy to use, free) icon set i've found -- but it's quite unwieldy to use without JS. if we can package it in a convenient way (maybe a CLI that downloads icon svgs) it can be the official icon pack of missing css - [Deniz](https://discord.com/channels/725789699527933952/941815579440992337/1372885158080020480)

> Check out the icônes website, it’s what I use to create my SVG spritesheets.
https://icones.js.org/collection/lucide - [ravenclaw900](https://discord.com/channels/725789699527933952/941815579440992337/1372913932079206522)

**Comment by geoffrey-eisenbarth** [comment]
> i really want there to be a way to use the icons that doesn't require (a) a build setup, (b) loading a massive blob of all the icons in the set or (c) individually downloading icons as needed, which really breaks my flow when building new pages. i don't know how to achieve that but i think it makes sense to build that under the missing.css umbrella - Deniz

For now, we'll just settle for recommending lucide and showing a (somewhat cumbersome) example
